go/vt/wrangler/testlib/backup_test.go

```diff
@@ -119,10 +115,7 @@ func testBackupRestore(t *testing.T, cDetails *compressionDetails) error {
 	backupstorage.BackupStorageImplementation = "file"
 	if cDetails != nil {
 		if cDetails.BuiltinCompressor != "" {
-			*mysqlctl.BuiltinCompressor = cDetails.BuiltinCompressor
-		}
-		if cDetails.BuiltinDecompressor != "" {
-			*mysqlctl.BuiltinDecompressor = cDetails.BuiltinDecompressor
+			*mysqlctl.CompressionEngineName = cDetails.BuiltinCompressor
```

Comment:
I thought `BuiltinCompressor` in `compressionDetails` has been renamed along with the flag. How is this working?

Reply:
Its a structure for unit test